react/components/Container/Container.css

```diff
@@ -0,0 +1 @@
+.container {}
```

Comment:
Useful?

Reply:
yes, it adds a CSS handle to allow customizing the component via CSS modules with the styles-builder